shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java

```diff
@@ -240,10 +240,10 @@ public TableRule getTableRule(final String logicTableName) {
     }
     
     /**
-     * Judge logic tables is all belong to binding encryptors.
+     * Judge whether all logic tables belongs to binding encryptors.
```

Comment:
@KSMASTERS What do you think of this? 

```java
/**
 * Judge whether all logic tables are binding encryptors or not.
 * 
 * @param logicTableNames logic table names
 * @return whether all logic tables are binding encryptors or not
 */
```

If you think it's good, please use similar expressions to modify the Java doc.